tests/pytests/test_aggregate.py

```diff
@@ -262,11 +262,11 @@ def testFilterBeforeLoad(self):
             # On cluster, filter can implicitly load any field
             res = self.env.cmd(*cmd)
             self.env.assertEqual([
-                ['categories', 'Accessories,Controllers,PC,Steering Wheels,Video Games', 'price', '759.12'],
-                ['categories', 'Consoles,Sony PSP,Video Games', 'price', '695.8'],
-                ['categories', 'Accessories,PC,Video Games', 'price', '599.99'],
-                ['categories', 'Accessories,Gaming Keyboards,Mac,Video Games', 'price', '559.99'],
-                ['categories', 'Consoles,Sony PSP,Video Games', 'price', '518.48']
+                ['price', '759.12', 'categories', 'Accessories,Controllers,PC,Steering Wheels,Video Games'],
+                ['price', '695.8', 'categories', 'Consoles,Sony PSP,Video Games'],
+                ['price', '599.99', 'categories', 'Accessories,PC,Video Games'],
+                ['price', '559.99', 'categories', 'Accessories,Gaming Keyboards,Mac,Video Games'],
```

Comment:
response schema changed due to shard explicit loading and order of load?

Reply:
Compared to current 2.8 yes. Compared to two days ago there couldn’t be any load after a filter, and any existing reference to a not-yet-referenced filed will remain in the same order it was